### PR TITLE
Add features.json

### PR DESCRIPTION
**Pull request by edd**
A new file for a new verison of `vegaprotocol/approbation` that maps specific criteria in to 'Features' so we can measure the tested-ness of a feature. This works independently from specs or categories.

- Add `features.json`

Note that I feel like both `categories.json` and `features.json` indicate to me that we're hitting the limits of the original design of approbation/the AC codes. What we're tracking now isn't quite what we decided to track when this started out. That's fine, but rather than bolt on any other optional things like this, let's plan AC coding v2 that better captures how we're planning to track development/testing in future.

### CODE REVIEW

File: protocol/features.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
...-VCBS-150", "0069-VCBS-151", "0069-VCBS-152", "0069-VCBS-153", "0069-VCBS-154", "0069-VCBS-155", "0069-VCBS-156", "0069-VCBS-157", "0069-VCBS-158", "0069-VCBS-159", "0069-VCBS-160", "0069-VCBS-161", "0069-VCBS-162", "0069-VCBS-163", "0069-VCBS-164", "0069-VCBS-165", "0069-VCBS-166", "0069-VCBS-167", "0069-VCBS-168", "0069-VCBS-169", "0069-VCBS-170", "0069-VCBS-171", "0069-VCBS-172", "0069-VCBS-173", "0069-VCBS-174", "0069-VCBS-175", "0069-VCBS-176", "0069-VCBS-177", "0070-MKTD-008", "0070-MKTD-009", "0070-MKTD-010", "0070-MKTD-011", "0070-MKTD-012", "0070-MKTD-013", "0070-MKTD-014", "0070-MKTD-015", "0073-LIMN-077", "0073-LIMN-078", "0073-LIMN-080", "0073-LIMN-082", "0073-LIMN-083", "0073-LIMN-084", "0073-LIMN-086", "0073-LIMN-087", "0073-LIMN-090", "0073-LIMN-091", "0073-LIMN-092", "0073-LIMN-093", "0073-LIMN-094", "0073-LIMN-095", "0073-LIMN-096", "0073-LIMN-097", "0073-LIMN-098", "0073-LIMN-099", "0073-LIMN-100", "0073-LIMN-101", "0073-LIMN-102", "0073-LIMN-103", "0073-LIMN-104", "0074-BTCH-011", "0074-BTCH-012", "0074-BTCH-013", "0074-BTCH-014", "0074-BTCH-015", "0074-BTCH-016", "0074-BTCH-017", "0074-BTCH-018", "0074-BTCH-019", "0074-BTCH-020", "0079-TGAP-006", "0079-TGAP-007"
```
Comment (gordsport) [inline protocol/features.json]: I dont think it should matter but should this be on a new line?

Comment (gordsport) [inline protocol/features.json]:
```suggestion
"0080-SPOT-001", "0080-SPOT-002", "0080-SPOT-003", "0080-SPOT-004", "0080-SPOT-005", "0080-SPOT-006", "0080-SPOT-007", "0080-SPOT-008", "0080-SPOT-009", "0080-SPOT-010", "0080-SPOT-011", "0080-SPOT-012", "0080-SPOT-013", "0080-SPOT-014", "0080-SPOT-015", "0080-SPOT-016", "0080-SPOT-017", "0080-SPOT-018", "0080-SPOT-019", "0080-SPOT-020", "0081-SUCM-004", "0044-LIME-043", "0044-LIME-044", "0044-LIME-045", "0044-LIME-046", "0044-LIME-047", "0044-LIME-048", "0056-REWA-051", "0056-REWA-052", "0056-REWA-053", "0056-REWA-054", "0056-REWA-055", "0056-REWA-056", "0056-REWA-057", "0056-REWA-058", "0056-REWA-059", "0056-REWA-060", "0056-REWA-061", "0056-REWA-062", "0056-REWA-063", "0056-REWA-064", "0056-REWA-065", "0056-REWA-066", "0056-REWA-067", "0056-REWA-068", "0056-REWA-069", "0056-REWA-070", "0056-REWA-071", "0056-REWA-072", "0056-REWA-073", "0056-REWA-074", "0056-REWA-075", "0073-LIMN-077", "0073-LIMN-078", "0073-LIMN-079", "0073-LIMN-080", "0073-LIMN-081", "0073-LIMN-082", "0073-LIMN-083", "0073-LIMN-084", "0073-LIMN-085", "0073-LIMN-086", "0073-LIMN-087", "0073-LIMN-088", "0073-LIMN-089", "0073-LIMN-090", "0073-LIMN-091", "0073-LIMN-092", "0073-LIMN-093", "0073-LIMN-094", "0073-LIMN-095", "0073-LIMN-096", "0073-LIMN-097", "0073-LIMN-098", "0073-LIMN-099", "0073-LIMN-100", "0037-OPEG-019", "0008-TRAD-008", "0004-AMND-030", "0004-AMND-031", "0004-AMND-032", "0004-AMND-033", "0004-AMND-034", "0004-AMND-035", "0004-AMND-036", "0004-AMND-037", "0004-AMND-038", "0004-AMND-039", "0004-AMND-040", "0004-AMND-041", "0004-AMND-042", "0004-AMND-043", "0004-AMND-044", "0004-AMND-045", "0004-AMND-046", "0004-AMND-047", "0004-AMND-048", "0004-AMND-049", "0004-AMND-050", "0004-AMND-051", "0004-AMND-052", "0004-AMND-053", "0004-AMND-054", "0004-AMND-055", "0011-MARA-018", "0011-MARA-019", "0011-MARA-020", "0011-MARA-021", "0011-MARA-022", "0011-MARA-023", "0011-MARA-024", "0011-MARA-025", "0011-MARA-026", "0011-MARA-027", "0011-MARA-028", "0011-MARA-029", "0011-MARA-030", "0011-MARA-031", "0011-MARA-032", "0013-ACCT-024", "0013-ACCT-025", "0013-ACCT-026", "0013-ACCT-027", "0013-ACCT-028", "0013-ACCT-029", "0013-ACCT-030", "0013-ACCT-031", "0014-ORDT-081", "0014-ORDT-082", "0014-ORDT-083", "0014-ORDT-084", "0014-ORDT-085", "0014-ORDT-086", "0014-ORDT-087", "0014-ORDT-088", "0014-ORDT-089", "0014-ORDT-090", "0014-ORDT-091", "0014-ORDT-092", "0014-ORDT-093", "0014-ORDT-094", "0014-ORDT-095", "0014-ORDT-096", "0014-ORDT-097", "0014-ORDT-098", "0014-ORDT-099", "0014-ORDT-100", "0014-ORDT-101", "0014-ORDT-102", "0014-ORDT-103", "0014-ORDT-104", "0014-ORDT-105", "0014-ORDT-106", "0014-ORDT-107", "0014-ORDT-108", "0014-ORDT-109", "0014-ORDT-110", "0014-ORDT-111", "0014-ORDT-112", "0014-ORDT-113", "0014-ORDT-114", "0014-ORDT-115", "0014-ORDT-116", "0014-ORDT-117", "0014-ORDT-118", "0014-ORDT-119", "0021-MDAT-013", "0021-MDAT-014", "0021-MDAT-015", "0021-MDAT-016", "0021-MDAT-017", "0021-MDAT-018", "0021-MDAT-019", "0021-MDAT-020", "0024-OSTA-030", "0024-OSTA-031", "0024-OSTA-032", "0024-OSTA-033", "0024-OSTA-034", "0024-OSTA-035", "0024-OSTA-036", "0024-OSTA-037", "0024-OSTA-038", "0024-OSTA-039", "0024-OSTA-040", "0024-OSTA-041", "0024-OSTA-042", "0024-OSTA-043", "0024-OSTA-044", "0024-OSTA-045", "0024-OSTA-046", "0024-OSTA-047", "0024-OSTA-048", "0025-OCRE-004", "0025-OCRE-005", "0025-OCRE-006", "0026-AUCT-023", "0026-AUCT-024", "0026-AUCT-025", "0026-AUCT-026", "0026-AUCT-027", "0026-AUCT-028", "0026-AUCT-029", "0026-AUCT-030", "0026-AUCT-031", "0026-AUCT-032", "0029-FEES-015", "0029-FEES-016", "0029-FEES-017", "0029-FEES-018", "0029-FEES-019", "0029-FEES-020", "0029-FEES-021", "0029-FEES-022", "0032-PRIM-022", "0032-PRIM-023", "0032-PRIM-024", "0032-PRIM-025", "0032-PRIM-026", "0032-PRIM-027", "0032-PRIM-028", "0032-PRIM-029", "0032-PRIM-030", "0032-PRIM-031", "0032-PRIM-032", "0032-PRIM-033", "0032-PRIM-034", "0032-PRIM-035", "0032-PRIM-036", "0032-PRIM-037", "0032-PRIM-038", "0033-OCAN-011", "0033-OCAN-012", "0033-OCAN-013", "0033-OCAN-014", "0033-OCAN-015", "0033-OCAN-016", "0033-OCAN-017", "0034-PROB-007", "0034-PROB-008", "0034-PROB-009", "0036-BRIE-004", "0036-BRIE-005", "0036-BRIE-006", "0039-MKTD-020", "0039-MKTD-021", "0039-MKTD-022", "0039-MKTD-023", "0039-MKTD-024", "0039-MKTD-025", "0039-MKTD-026", "0039-MKTD-027", "0039-MKTD-028", "0039-MKTD-029", "0039-MKTD-030", "0039-MKTD-031", "0039-MKTD-032", "0039-MKTD-033", "0043-MKTL-005", "0043-MKTL-006", "0043-MKTL-007", "0043-MKTL-008", "0049-TVAL-007", "0049-TVAL-008", "0049-TVAL-009", "0049-TVAL-010", "0049-TVAL-011", "0049-TVAL-012", "0051-PROD-004", "0051-PROD-005", "0051-PROD-006", "0052-FPOS-003", "0052-FPOS-004", "0054-NETP-007", "0054-NETP-008", "0054-NETP-009", "0054-NETP-010", "0054-NETP-011", "0057-TRAN-063", "0065-FTCO-005", "0065-FTCO-006", "0065-FTCO-007", "0065-FTCO-008", "0066-VALW-004", "0066-VALW-005", "0068-MATC-060", "0068-MATC-061", "0068-MATC-062", "0068-MATC-063", "0068-MATC-064", "0068-MATC-065", "0068-MATC-066", "0068-MATC-067", "0068-MATC-068", "0068-MATC-069", "0068-MATC-070", "0068-MATC-071", "0068-MATC-072", "0068-MATC-073", "0068-MATC-074", "0068-MATC-075", "0068-MATC-076", "0068-MATC-077", "0068-MATC-078", "0068-MATC-079", "0068-MATC-080", "0068-MATC-081", "0068-MATC-082", "0068-MATC-083", "0068-MATC-084", "0068-MATC-085", "0068-MATC-086", "0068-MATC-087", "0068-MATC-088", "0068-MATC-089", "0068-MATC-090", "0068-MATC-091", "0068-MATC-092", "0069-VCBS-089", "0069-VCBS-090", "0069-VCBS-091", "0069-VCBS-092", "0069-VCBS-093", "0069-VCBS-094", "0069-VCBS-095", "0069-VCBS-096", "0069-VCBS-097", "0069-VCBS-098", "0069-VCBS-099", "0069-VCBS-100", "0069-VCBS-101", "0069-VCBS-102", "0069-VCBS-103", "0069-VCBS-104", "0069-VCBS-105", "0069-VCBS-106", "0069-VCBS-107", "0069-VCBS-108", "0069-VCBS-109", "0069-VCBS-110", "0069-VCBS-111", "0069-VCBS-112", "0069-VCBS-113", "0069-VCBS-114", "0069-VCBS-115", "0069-VCBS-116", "0069-VCBS-117", "0069-VCBS-118", "0069-VCBS-119", "0069-VCBS-120", "0069-VCBS-121", "0069-VCBS-122", "0069-VCBS-123", "0069-VCBS-124", "0069-VCBS-125", "0069-VCBS-126", "0069-VCBS-127", "0069-VCBS-128", "0069-VCBS-129", "0069-VCBS-130", "0069-VCBS-131", "0069-VCBS-132", "0069-VCBS-133", "0069-VCBS-134", "0069-VCBS-135", "0069-VCBS-136", "0069-VCBS-137", "0069-VCBS-138", "0069-VCBS-139", "0069-VCBS-140", "0069-VCBS-141", "0069-VCBS-142", "0069-VCBS-143", "0069-VCBS-144", "0069-VCBS-145", "0069-VCBS-146", "0069-VCBS-147", "0069-VCBS-148", "0069-VCBS-149", "0069-VCBS-150", "0069-VCBS-151", "0069-VCBS-152", "0069-VCBS-153", "0069-VCBS-154", "0069-VCBS-155", "0069-VCBS-156", "0069-VCBS-157", "0069-VCBS-158", "0069-VCBS-159", "0069-VCBS-160", "0069-VCBS-161", "0069-VCBS-162", "0069-VCBS-163", "0069-VCBS-164", "0069-VCBS-165", "0069-VCBS-166", "0069-VCBS-167", "0069-VCBS-168", "0069-VCBS-169", "0069-VCBS-170", "0069-VCBS-171", "0069-VCBS-172", "0069-VCBS-173", "0069-VCBS-174", "0069-VCBS-175", "0069-VCBS-176", "0069-VCBS-177", "0070-MKTD-008", "0070-MKTD-009", "0070-MKTD-010", "0070-MKTD-011", "0070-MKTD-012", "0070-MKTD-013", "0070-MKTD-014", "0070-MKTD-015", "0073-LIMN-077", "0073-LIMN-078", "0073-LIMN-080", "0073-LIMN-082", "0073-LIMN-083", "0073-LIMN-084", "0073-LIMN-086", "0073-LIMN-087", "0073-LIMN-090", "0073-LIMN-091", "0073-LIMN-092", "0073-LIMN-093", "0073-LIMN-094", "0073-LIMN-095", "0073-LIMN-096", "0073-LIMN-097", "0073-LIMN-098", "0073-LIMN-099", "0073-LIMN-100", "0073-LIMN-101", "0073-LIMN-102", "0073-LIMN-103", "0073-LIMN-104", "0074-BTCH-011", "0074-BTCH-012", "0074-BTCH-013", "0074-BTCH-014", "0074-BTCH-015", "0074-BTCH-016", "0074-BTCH-017", "0074-BTCH-018", "0074-BTCH-019", "0074-BTCH-020", "0079-TGAP-006", "0079-TGAP-007"
```

Comment (gordsport) [reply protocol/features.json]: I have also checked the codes updated in the suggested changes (`PPROD` and `BBRIE`) are not present in the specs repo PR: https://github.com/vegaprotocol/specs/pull/1791

Comment (edd) [reply protocol/features.json]: It doesn’t matter but it should be on a single line for consistency. Fixed.

**Review by gordsport — APPROVED** [review]
LGTM

**Pull request merged**